dotnet/src/VectorDataIntegrationTests/AzureAISearchIntegrationTests/CRUD/AzureAISearchAllSupportedTypesTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using AzureAISearchIntegrationTests.Support;
+using Microsoft.Extensions.VectorData;
+using VectorDataSpecificationTests.Xunit;
+using Xunit;
+
+namespace AzureAISearchIntegrationTests.CRUD;
+
+public class AzureAISearchAllSupportedTypesTests(AzureAISearchFixture fixture) : IClassFixture<AzureAISearchFixture>
+{
+    [ConditionalFact]
+    public async Task AllTypesBatchGetAsync()
```

Comment:
I think we need to have general type tests, i.e. a conformance test suite for all providers where all aspects of a type are tested end-to-end (rather than a provider-specific test as here); an example would be EmbeddingTypeTests, which provides infra for providers to do end-to-end testing. BTW the below tests Upsert and Get (so serialization/deserialization), but not filter support which we should also do.

But of course, no problem merging this for now and improving later.